### PR TITLE
Address comments - EverythingLM

### PR DESCRIPTION
**Pull request by manandey**
Hi @shayne-longpre @Muennighoff,

I have tried to address the following comments and the tests pass as well. Feel free to let me know if any other changes are required.

> - Could we retroactively add the OpenAI license/terms to the list of licenses
> - It looks like in HF they have a pie chart with a few more tasks we didn't include in the Task Categories. Any chance we could expand our list to make sure its covering all of there's?

Thanks,
Manan

### CODE REVIEW

File: data_summaries/EverythingLM.json [hunk 1]
```diff
@@ -35,6 +45,10 @@
             {
                 "License": "MIT License",
                 "License URL": "https://huggingface.co/datasets/totally-not-an-llm/EverythingLM-data-V3"
+            },
+            {
+                "License": "OpenAI",
+                "License URL": "https://huggingface.co/datasets/totally-not-an-llm/EverythingLM-data-V3"
```
Comment (Muennighoff) [inline data_summaries/EverythingLM.json:51]: Can you change this to the OAI URL? cc @shayne-longpre

Comment (manandey) [reply data_summaries/EverythingLM.json:51]: Sure, but in other PR's I saw people using the respective dataset's licence or URL itself. e.g. 

- https://github.com/Data-Provenance-Initiative/Data-Provenance-Collection/blob/004761178ea7b9ea9c542d081296edc5e66bc314/data_summaries/Alpaca.json#L53
- https://github.com/Data-Provenance-Initiative/Data-Provenance-Collection/blob/004761178ea7b9ea9c542d081296edc5e66bc314/data_summaries/GPTeacher.json#L51
- https://github.com/Data-Provenance-Initiative/Data-Provenance-Collection/blob/004761178ea7b9ea9c542d081296edc5e66bc314/data_summaries/UltraChat.json#L57

Also, do you want me to use the specific OAI model's URL?

Comment (shayne-longpre) [reply data_summaries/EverythingLM.json:51]: I think this is fine -- we have the OAI URL encoded elsewhere that we can always link to. It's nice to see where they specify that they use OAI to generate the data so we can infer that the license is accurate.

**Review by shayne-longpre — APPROVED** [review]
LGTM! Feel free to merge -- just run the tests to double check nothing is broken!

**Comment by shayne-longpre** [comment]
LGTM

**Pull request merged**